tests/test_logger.py

```diff
@@ -177,7 +177,7 @@ def test_custom_logging_config_is_parsed_and_used_when_provided():
                    logging_config_file.name), patch(
                        "logging.config.dictConfig") as dict_config_mock:
             _configure_vllm_root_logger()
-            assert dict_config_mock.called_with(valid_logging_config)
+            dict_config_mock.assert_called_with(valid_logging_config)
```

Comment:
This is failing and I'm not sure what the expected behavior of this test was before, looking into it a bit to try to understand

Reply:
So I think that the mock on `logging.config.dictConfig` was incorrect, because `vllm.logger` imports that eagerly, so we instead have to mock the local `vllm.logger.dictConfig`.

The test was passing before because `dict_config_mock.called_with` was returning a mock, which was then passing the `assert` since it's not falsey.